### DISCUSSION TITLE
How many devices?

### DISCUSSION THREAD

**Discussion by TorbenMei** [Koenkk/zigbee2mqtt#18571] [Devices]
Does anyone have experience with how many devices you can have running on your Zigbee2MQTT-system?
I am approaching 100 devices and find that the system has become more unstable - e.g. several devices have gone "offline", however, some of them are still working and were last seen 1 munte ago.
I am useing "SONOFF Zigbee 3.0 USB Dongle Plus (E version)", RPi4, Openhabian
/Torben

**Comment by lux73** [comment]
as you can see [here](https://www.zigbee2mqtt.io/guide/adapters/#experimental) SONOFF Dongle E is declared as experimental and not recommend for stable Environment

please buy another recommended Coordinator
